patches/@react-navigation+stack+6.3.16+003+fixKeyboardFlicker.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@react-navigation/stack/src/views/Stack/Card.tsx b/node_modules/@react-navigation/stack/src/views/Stack/Card.tsx
+index 913218e..ebc2f93 100755
+--- a/node_modules/@react-navigation/stack/src/views/Stack/Card.tsx
++++ b/node_modules/@react-navigation/stack/src/views/Stack/Card.tsx
+@@ -517,7 +517,7 @@ export default class Card extends React.Component<Props> {
+           // Make sure that this view isn't removed. If this view is removed, our style with animated value won't apply
+           collapsable={false}
+         />
+-        <View pointerEvents="box-none" {...rest}>
++        <View pointerEvents="box-none" collapsable={false} {...rest}>
```

Comment:
This was merged upstream and released in `6.3.29`. Would be great to upgrade that and remove this patch, ideally in a separate PR

Reply:
Here is the PR: https://github.com/Expensify/App/pull/40008